### PR TITLE
Cronitor plugin

### PR DESCRIPTION
**Pull request by liiight**
### Motivation for changes:
Wanted an uptime monitor. Cronitor is simple and free
### Detailed changes:
- Added cronitor plugin

```yaml
cronitor: ABC123
```
Or:
```yaml
cronitor:
  monitor_code: ABC123
  on_start: yes
  on_abort: no
  auth_key: FOOBAR
```

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -21,7 +21,6 @@ flask-login==0.4.0
 flask-restful==0.3.6
 flask-restplus==0.10.1
 flask==1.0.2
-future==0.16.0
```
Comment (cvium) [inline requirements.txt]: Hmm?

Comment (liiight) [reply requirements.txt]: Slippage

File: flexget/components/notify/notifiers/cronitor.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
+from builtins import *
```
Comment (cvium) [inline flexget/components/notify/notifiers/cronitor.py]: Missing future? Absolute import etc.

File: flexget/components/notify/notifiers/cronitor.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                },
+                "required": ["monitor_code"],
+                "additionalProperties": False,
+            },
+            {"type": "string"},
+        ]
+    }
+
+    def __init__(self):
+        self.config = None
```
Comment (paranoidi) [inline flexget/components/notify/notifiers/cronitor.py]: This is not good practice as it leaves (possibly) behind all kinds of objects after plugin execution has finished.

Comment (liiight) [reply flexget/components/notify/notifiers/cronitor.py]: it's either this or prepare config and pass it for every task execution phase. any recommendation?

File: flexget/components/notify/notifiers/cronitor.py (hunk 2 above)
Comment (liiight) [reply flexget/components/notify/notifiers/cronitor.py]: not sure what you mean

Comment (gazpachoking) [reply flexget/components/notify/notifiers/cronitor.py]: `from __future__ import unicode_literals...` I think he means.

File: requirements.txt [hunk 4]
```diff
@@ -30,7 +30,6 @@ itsdangerous==0.24        # via flask
 jinja2==2.10
 jsonschema==2.6.0
 markupsafe==1.0           # via jinja2
-path.py==11.5.0
```
Comment (gazpachoking) [inline requirements.txt]: Do we not need path.py anymore? Did this leak over from python 3 only branch?

Comment (liiight) [reply requirements.txt]: yeah...

**Pull request merged**